src/Vault.sol

```diff
@@ -288,14 +303,13 @@ contract Vault {
      * collateralised assets can fluctuate, the haircut guarantees that the vault
      * remains over-collateralised with a high confidence level (99,9%+). The size of the
      * haircut depends on the underlying risk of the assets in the vault, the bigger the volatility
-     * or the smaller the on-chain liquidity, the biggert the haircut will be.
+     * or the smaller the on-chain liquidity, the bigger the haircut will be.
      */
     function getCollateralValue() public view returns (uint256 collateralValue) {
-        //gas: cannot overflow unless currentValue is more than
-        // 1.15**57 *10**18 decimals, which is too many billions to write out
-        unchecked {
-            collateralValue = getVaultValue(vault.baseCurrency) * 100 / vault.collThres;
-        }
+        (address[] memory assetAddresses, uint256[] memory assetIds, uint256[] memory assetAmounts) =
```

Comment:
Variables not used anymore

Reply:
They are used in Registry getCollateralValue call?

Reply:
Comment was for line 399, not 309, apologies